bigbluebutton-html5/imports/ui/components/actions-bar/container.jsx

```diff
@@ -31,10 +33,19 @@ import { withModalMounter } from '/imports/ui/components/modal/service';
 
 const ActionsBarContainer = props => <ActionsBar {...props} />;
 const POLLING_ENABLED = Meteor.settings.public.poll.enabled;
+const { dataSaving } = Settings;
+const { viewParticipantsWebcams, viewScreenshare } = dataSaving;
 
 export default withModalMounter(withTracker(({ mountModal }) => ({
   amIPresenter: Service.amIPresenter(),
   amIModerator: Service.amIModerator(),
+  presenter: Service.getPresenter(),
+  swapLayout: (getSwapLayout()) && shouldEnableSwapLayout(),
+  usersVideo: VideoService.getAllWebcamUsers(),
+  singleWebcam: (VideoService.getAllWebcamUsers().length < 2),
```

Comment:
why do you need these?

Reply:
Removed these properties except getPresenter()